lms/static/scripts/frontend_apps/components/BookSelector.tsx

```diff
@@ -194,23 +193,15 @@ export default function BookSelector({
         </InputGroup>
 
         {selectedBook && (
-          <div
-            className="flex flex-row items-center space-x-2"
-            data-testid="selected-book"
-          >
-            <CheckIcon className="text-green-success" />
-            <div className="grow font-bold italic">{selectedBook.title}</div>
-          </div>
+          <UIMessage status="success" data-testid="selected-book">
+            <span className="font-bold italic">{selectedBook.title}</span>
```

Comment:
In the current set of uses, we always set bold and italic on success messages, but this was flexibility I wanted to leave to the consumer a d not bake into the styling of the messages.